`make clean` doesn't work

```
$ make clean
Makefile:104: warning: overriding recipe for target 'ctl'
Makefile:77: warning: ignoring old recipe for target 'ctl'
make: *** No rule to make target 'clean'.  Stop.
```

/kind feature

--- a/Makefile
+++ b/Makefile
@@ -57,6 +57,9 @@ help:
 # Alias targets
 ###############
 
+clean:
+	bazel clean --expunge
+
 build: ctl images
 
 verify:

--- a/Makefile
+++ b/Makefile
@@ -38,6 +38,7 @@ help:
 	#
 	### Build targets
 	#
+	# clean              - removes the entire output base tree and stop the Bazel server
 	# controller         - build a binary of the 'controller'
 	# cainjector         - build a binary of the 'cainjector'
 	# webhook            - build a binary of the 'webhook'

